main.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import FastAPI
+
+from city import router as city_router
+from temperature import router as temperature_router
+
+
+app = FastAPI()
+
+app.include_router(city_router.router)
+app.include_router(temperature_router.router)
+# EOF
```

Comment:
EOF is an empty line at the end of file. Remove commented code and add just empty line.